app/views/home/firstEditionProjects.html.slim

```diff
@@ -0,0 +1,116 @@
+h1 class="text-center LITG-green" "Learn IT, Girl" first edition projects
+div class="row"
+  div class="col-md-6 col-md-offset-3"
```

Comment:
it's okay :) just for the future reference instead of writing `h1 class="text-center LITG-green"` you can type `h1.text-center.LITG-green` and `div class="col-md-6 col-md-offset-3"` can be replaced with `.col-md-6.col-md-offset-3`